addons/website/views/snippets/snippets.xml

```diff
@@ -309,6 +309,27 @@
 </template>
 
 <template id="snippet_options">
+    <div data-selector="#wrapwrap > header nav.navbar .o_logo" data-no-check="true">
+        <we-select string="Type" data-reload="/">
+            <we-button data-customize-website-views="" data-name="option_header_brand_none">None</we-button>
+            <we-button data-customize-website-views="website.option_header_brand_name">Text</we-button>
+            <we-button data-customize-website-views="website.option_header_brand_logo">Image</we-button>
+        </we-select>
+        <we-input string="⌙ Height"
```

Comment:
Does it make sense to keep the `⌙` if the options are now in their own section about the logo?